libs/httpproxy/proxy-camel-route/pom.xml

```diff
@@ -77,6 +77,11 @@
             <artifactId>common-system</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
❓ Didn't we just got rid of findbugs?

Reply:
Yes. There are still some left-overs in the POM files. We used to depend on findbugs for the @Nullable annotation. I removed this dependency and added `org.apache.servicemix.bundles.jsr305` instead.